README.md

```diff
@@ -263,8 +263,8 @@ The ForageSDK exposes the following function to collect the EBT card number:
 ```kotlin
     data class TokenizeEBTCardParams(
         val foragePANEditText: ForagePANEditText,
-        val customerId: String,
-        val reusable: Boolean
+        val customerId: String?,
+        val reusable: Boolean?
```

Comment:
This was also made optional in #151

Reply:
Just a callout to be careful when merging this change in after #151 to make sure nothing is weird/overwritten